.travis.yml

```diff
@@ -11,8 +11,8 @@ python:
 
 env:
   - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" INCLUDE="woocommerceerpconnect"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" INCLUDE="connector_woocommerce"
```

Comment:
It is a good to check minimal compatibility between modules.
You can also have each module individually tested using `UNIT_TEST="1"` (although redundant when you have only one module in the repo).

You may try this:

```
env:
  global:
  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"

  matrix:
  - LINT_CHECK="1"
  - TESTS="1" ODOO_REPO="odoo/odoo"
  - TESTS="1" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
  - TESTS="1" ODOO_REPO="OCA/OCB"
```